firefox_ui_harness/runtests.py

```diff
@@ -87,9 +44,10 @@ def startTestRunner(runner_class, options, tests):
 
 
 def cli(runner_class=FirefoxUITestRunner, parser_class=FirefoxUIOptions):
-    parser = parser_class(usage='%prog [options] test_file_or_dir <test_file_or_dir> ...')
+    parser = parser_class(usage='%prog [options] manifest <test_file_or_dir> ...')
```

Comment:
The cli is actually something like "firefox-ui-tests [options] manifest_or_test_file", right?